_layouts/release.html.haml

```diff
@@ -67,7 +67,7 @@
           %a.artifact{:href=>a.to_javadoc_url, :target=>'_blank'} javadoc
         %a.artifact{:href=>a.to_pom_url, :target=>'_blank'} pom
 - unless page.release_notes.nil?
-  %h3
+  %h3.release-notes
```

Comment:
I'm slightly confused - I haven't found any of the used classes/styles in css `release-details` `published-artifacts` `release-notes`. If there isn't any, why do you use them?